### DISCUSSION TITLE
Make all boosting items not work underwater

### DISCUSSION THREAD

**Discussion by Fluff8836** [DoukutsuRandamu/IssueTracker#113] [Suggestions]
Those equipped items that take priority over boosting are items that allow for double jumping, they are Auxiliary Booster, Alien Medal, Amphibian Medal and Cape of Shadows.

And when player is underwater, they can press jump repeatedly for multiple double jumping, which is swimming, however that **does not take priority** over boosting.

Make underwater swimming take priority over boosting items.

Boosting items are Booster 2.0, Energy Canister, Super Booster & Terra Boots.

**Comment by TrashboxBobylev** [comment]
Done in 1.44.2.
